app/views/katello/subscription_mailer/subscription_expiry.html.erb

```diff
@@ -41,5 +41,6 @@
       <% end %>
     </table>
 </div>
+<br> <%= link_to _("View the full report"), report_url %>
```

Comment:
I think the tag should be `<br/>` and the link (@report_link per my other feedback) on a new line